server/src/controllers/tagController.js

```diff
@@ -12,21 +12,21 @@ const addJobTags = (req, res) => {
       id: job.id
     }
   })
-    .then( (foundJob) => {
-      tagsData.forEach((tag) => {
-        Tag.findOrCreate({ 
-          where: { name: tag }
-        })
-        .spread((tag, created) => {
-          //Sequelize association that adds job and tag to JobTag join table
-          tag.addJob(foundJob)
-          res.end();
-        })
+  .then((foundJob) => {
+    tagsData.forEach((tag) => {
+      Tag.findOrCreate({ 
+        where: { name: tag }
+      })
+      .spread((tag, created) => {
+        //Sequelize association that adds job and tag to JobTag join table
+        tag.addJob(foundJob)
+        res.end();
```

Comment:
Good catch, I think this is a mistake, the end should happen once all tags are created.